src/QsCompiler/SyntaxProcessor/SyntaxExtensions.fs

```diff
@@ -18,222 +18,55 @@ open Microsoft.Quantum.QsCompiler.SyntaxTokens
-    | QsExpressionKind.BAND (lhs, rhs) -> ([ lhs; rhs ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.BXOR (lhs, rhs) -> ([ lhs; rhs ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.LSHIFT (lhs, rhs) -> ([ lhs; rhs ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.RSHIFT (lhs, rhs) -> ([ lhs; rhs ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.CopyAndUpdate (lhs, acc, rhs) -> ([ lhs; rhs; acc ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.CONDITIONAL (cond, case1, case2) -> ([ cond; case1; case2 ], []) |> collectWith SymbolsFromExpr
-    | QsExpressionKind.MissingExpr -> [], [], [ item ]
-    | QsExpressionKind.InvalidExpr -> [], [], [ item ]
-
 let private AttributeAsCallExpr (sym: QsSymbol, ex: QsExpression) =
     let combinedRange = QsNullable.Map2 Range.Span sym.Range ex.Range
     let id = { Expression = QsExpressionKind.Identifier(sym, Null); Range = sym.Range }
     { Expression = QsExpressionKind.CallLikeExpression(id, ex); Range = combinedRange }
 
-let rec private SymbolDeclarations (sym: QsSymbol) =
-    match sym.Symbol with
-    | SymbolTuple items ->
-        [
-            for item in items do
-                yield item
-        ]
-        |> List.collect SymbolDeclarations
-    | InvalidSymbol
-    | MissingSymbol -> []
-    | _ -> [ sym ]
-
-let private SymbolsInGenerator (gen: QsSpecializationGenerator) =
-    match gen.Generator with
-    | UserDefinedImplementation sym -> sym |> SymbolDeclarations
-    | _ -> []
-
-let private SymbolsInArgumentTuple (declName, argTuple) =
-    let recur extract items =
-        let extracted : (QsSymbol list * QsType list) list = items |> List.map extract
-
-        let getSeq select =
-            [
-                for seq in extracted do
-                    for item in select seq do
-                        yield item
-            ]
-
-        getSeq fst, getSeq snd
-
-    let rec extract =
-        function
-        | QsTuple items ->
-            [
-                for item in items do
-                    yield item
-            ]
-            |> recur extract
-        | QsTupleItem (sym, t) -> sym |> SymbolDeclarations, t |> TypeNameSymbols
-
-    let decl, types = argTuple |> extract
-
-    List.concat [ declName |> SymbolDeclarations
-                  decl ],
-    ([], types, [])
-
-let private SymbolsInCallableDeclaration (name: QsSymbol, signature: CallableSignature) =
-    let symDecl, (vars, types, exs) = SymbolsInArgumentTuple(name, signature.Argument)
-
-    let typeParams =
-        let build (sym: QsSymbol) =
-            { Type = QsTypeKind.TypeParameter sym; Range = sym.Range }
-
-        [
-            for param in signature.TypeParameters do
-                yield build param
-        ]
-
-    symDecl,
-    (vars,
-     List.concat [ typeParams
-                   types
-                   signature.ReturnType |> TypeNameSymbols ],
-     exs)
-
+// TODO: Remove after 2022-05 release.
```

Comment:
Could you please follow the patter used for other such todos, so that they are easier to find in the code:
TODO: RELEASE 2022-05: Remove SymbolInformation.